clinical/dbaccess.py

```diff
@@ -13,112 +13,6 @@
 import psutil
 import subprocess
 
-def readconfig( config ):
-  """Reads parameters from a config file.
-
-  Args:
-    config (str): path to config file
-    If config does not exist the default will be used
-  
-  Returns:
-    dict: parameters from the config file (unparsed)
-  """
-  if os.path.isfile(config):
-    Configfile = config
-  else:
-    Configfile = "/home/hiseq.clinical/.scilifelabrc"
-  Params = {}
-  with open(Configfile, "r") as Confs:
-    for line in Confs:
-      if len(line) > 5 and not line[0] == "#":
-        line = line.rstrip()
-        pv = line.split(" ")
-        if len(pv) > 1:
-          arg = pv[0]
-          pv.pop(0)
-          Params[arg] = ' '.join(pv)
-        else:
-          Params[pv[0]] = pv[1]
-  return Params
-
-def create_tunnel(tunnel_cmd):
-  ssh_process = subprocess.Popen(tunnel_cmd,  universal_newlines=True,
-                                                shell=True,
-                                                stdout=subprocess.PIPE,
-                                                stderr=subprocess.STDOUT,
-                                                stdin=subprocess.PIPE)
-    # Assuming that the tunnel command has "-f" and "ExitOnForwardFailure=yes", then the 
-    # command will return immediately so we can check the return status with a poll().
-  while True:
-    p = ssh_process.poll()
-    if p is not None: break
-    time.sleep(1)
-  if p == 0:
```

Comment:
This block might be possible to factor out into a separate function that has the job of finding a matching process. It seems that would make the code easier to test and that's usually a hint I find good to think about :smiley: